plugins/@grouparoo/spec-helper/src/lib/specHelper.ts

```diff
@@ -181,6 +181,93 @@ export namespace helper {
     }
   }
 
+  /**
+   * Run a Grouparoo server "for real" in a sub-process
+   */
+
+  export const startGrouparooTestServerDetached = async ({
+    port,
+    truncate = false,
+    resetSettings = false,
+    runMode = "cli:start",
+  }: {
+    port: number | string;
+    truncate?: boolean;
+    resetSettings?: boolean;
+    runMode?: GrouparooRunMode;
+  }): Promise<ChildProcessWithoutNullStreams> => {
+    let serverProcess: ChildProcessWithoutNullStreams;
+
+    if (truncate || resetSettings) {
```

Comment:
That's an extreme way to reset the settings for sure!

Reply:
It's the same as the old implementation of the Detached Test Server.  Just not in a `beforeAll`